pkg/pgmodel/extension.go

```diff
@@ -117,22 +150,25 @@ func fetchAvailableExtensionVersions(conn *pgxpool.Pool) (semver.Versions, error
 	return versions, nil
 }
 
-func fetchInstalledExtensionVersion(conn *pgxpool.Pool) (semver.Version, bool, error) {
+func fetchInstalledExtensionVersion(conn *pgxpool.Pool, extensionName string) (semver.Version, bool, error) {
 	var versionString string
-	err := conn.QueryRow(context.Background(),
-		"SELECT extversion  FROM pg_extension WHERE extname='timescale_prometheus_extra'").Scan(&versionString)
-	if err != nil {
+	if err := conn.QueryRow(
+		context.Background(),
+		fmt.Sprintf("SELECT extversion FROM pg_extension WHERE extname='%s';", extensionName),
```

Comment:
```suggestion
		"SELECT extversion FROM pg_extension WHERE extname=$1", extensionName,
```
Using parameters is better practice. Among other things its safer again SQL-injection attacks. Sometimes we can't use it, but we should try to use parameters whenever we can.

Reply:
Ahh, got it.